testing/test-utils/src/lib/utils/nx.ts

```diff
@@ -0,0 +1,23 @@
+import { ExecutorContext } from '@nx/devkit';
+
+export function executorContext(
+  nameOrOpt: string | { projectName: string; cwd?: string } = 'my-lib',
+): ExecutorContext {
+  const { projectName, cwd = '' } =
```

Comment:
Is an empty string a good default for `cwd`? I would expect `process.cwd()`, or leaving it undefined.

Reply:
fixed! left it undefined and adjusted the type